index.html

```diff
@@ -0,0 +1,339 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <section class="menu-mobile">
+      <div id="menu-container" class="mobile-menu-container">
+        <div class="bg-img"></div>
+        <button class="close-btn">x</button>
+        <ul class="mobile-menu-ul" id="anchor-tags">
+          <li><a href="#works">Portfolio</a></li>
+          <li><a href="#about-myself">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+    </section>
+    <header class="main-header">
+      <nav>
+        <ul class="header-menu">
+          <li><a href="#" class="info-text">Melkamu</a></li>
+          <li>
+            <a href="#" id="hamburger-btn" class="nav__button"
+              ><i
+                ><img
+                  src="Asset/Icon - Menu.png"
+                  alt="hamburger-icon-menu"
+                  class="info-image"
+                  id="hamburger-btnicon" /></i
+            ></a>
+          </li>
+          <li class="desktop-menu-container">
+            <ul class="menu-lists">
+              <li class="item"><a href="#portfolio">Portfolio</a></li>
+              <li class="item"><a href="#about">About</a></li>
+              <li class="item"><a href="#contact">Contact</a></li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-content">
+      <section class="headlines">
+        <div class="container">
+          <div class="about" id="about-info">
+            <h1 class="intro">I’m Melkamu.<br />Glad to see you!</h1>
+            <p>
+              I'm a software developer! I can help you build a product, feature,
+              or website. Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.